### PR TITLE
Refactor/more reducers

### PR DESCRIPTION
**Pull request by JiaqiWang18**
## Description
Refactor the remaining reducers
## Change Log
Refactored the below
`optional_courses_reducer.jsx`
`peer_modal_reducer.jsx`
`integration_modal_reducer.jsx`
`friends_reducer.jsx`
`notification_token_reducer.jsx`
`entities_reducer.jsx`

Deleted `integrations_reducer` because its data is not used anywhere

### CODE REVIEW

**Comment by JJamesWWang** [comment]
Hmm if integrations reducer isn't used, does that mean we can delete the Integration model?

**Comment by JiaqiWang18** [comment]
> Hmm if integrations reducer isn't used, does that mean we can delete the Integration model?

It seems like integration_modal_reducer also contains some integration data. Maybe those aren't used as well.

File: timetable/templates/index.html [hunk 1]
```diff
@@ -275,7 +275,7 @@ <h3>Select your school to begin</h3>
 						</a></li>
 				</ul>
 			</div>
-			<h6>COPYRIGHT © 2017-2022 SEMESTERLY TECHNOLOGIES, LLC.</h6>
+			<h6>COPYRIGHT © 2017 SEMESTERLY TECHNOLOGIES, LLC.</h6>
```
Comment (JJamesWWang) [inline timetable/templates/index.html]: Do you know why it's deleting the -2022 edits

Comment (JiaqiWang18) [reply timetable/templates/index.html]: Not sure, I merged Sebastian's change so I think he changed this file?

Comment (JiaqiWang18) [reply timetable/templates/index.html]: #657

Comment (JJamesWWang) [reply timetable/templates/index.html]: huh, okay

**Review by JJamesWWang — APPROVED** [review]
Looks good, maybe we can look into deleting integrations_modal and then the Integration model

**Pull request merged**